docs/faq.md

```diff
@@ -155,6 +155,18 @@ CNAMEs cannot co-exist with other records, therefore you can use the `--txt-pref
 
 You need to add either https://www.googleapis.com/auth/ndev.clouddns.readwrite or https://www.googleapis.com/auth/cloud-platform on your instance group's scope.
 
+### What metrics can I get from ExternalDNS and what do they mean?  
+
+ExternalDNS exposes 2 types of metrics: Sources and Registry errors.
+
+`Source`s are mostly Kubernetes API objects. Examples of `source` errors may be connection errors to the Kubernetes API server itself or missing RBAC permissions. It can also stem from incompatible configuration in the objects itself like invalid characters, processing a broken fqdnTemplate etc.
+
+`Registry` errors are mostly Provider errors, unless there's some coding flaw in the registry package. Provider errors often arise due to accessing their APIs due to network or missing cloud-provider permissions when reading records. When applying a changeset, errors arise when the changeset applied is incompatible with the current state, in which case ExternalDNS can get stuck forever.
```

Comment:
I would clarify this sentence. What does it mean to be "stuck forever"? If I read it like that I would rather think of a bug, not something that should be covered by FAQ. Can you try to rephrase with something a bit clearer?

Reply:
Sure -- in that case I would just remove the part about ExternalDNS getting stuck